packages/account-v2/src/pages/LoginHistory/LoginHistoryTable.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { LoginHistory } from '@deriv/api-types';
+import { Table } from '@deriv-com/ui';
+import { useFormattedLoginHistoryData } from '../../hooks/useFormattedLoginHistoryData';
+
+type TLoginHistoryData = {
+    loginHistory: LoginHistory | undefined;
```

Comment:
When do we have loginHistory as undefined. if  loginHistory is undefined, isnt iot better not to render this component?

Reply:
have removed 
```
type TLoginHistoryData = {
    loginHistory: LoginHistory | undefined;
    }
```
and the code works the same but for the undefined conditions 

but on using the undefind condition in the imag below it gives the value 0 so what is to be done please if you could suggest ?

![Screenshot 2024-03-25 at 3 49 40 PM](https://github.com/binary-com/deriv-app/assets/158162395/5dd1b73f-ffef-42b1-b1c0-29cecbb79217)

<img width="1716" alt="Screenshot 2024-03-25 at 3 49 55 PM" src="https://github.com/binary-com/deriv-app/assets/158162395/b7e87282-c8b0-42da-8260-c3a89b136bae">